environments/vm/secrets/vm.yml

```diff
@@ -26,6 +26,9 @@ engine_janus_secret: secret
 
 engine_ldap_password: secret
 
+# Minimum execution time in milliseconds when response is invalid
+engine_minimum_execution_time_on_invalid_response: 5000
```

Comment:
What do you mean by 'this'? Regarding placement in the secrets file, see my general comment above. We could also hard-code it in `engineblock.ini.j2`, but that would make it less transparently configurable IMHO.